imod_coupler/drivers/dfm_metamod/dfm_utilities.py

```diff
@@ -0,0 +1,27 @@
+import numpy as np
+from bmi.wrapper import BMIWrapper
+from numpy.typing import NDArray
+
+
+class DfmUtilities:
+    @classmethod
+    def get_number_1d_nodes(cls, dflow: BMIWrapper) -> int:
+        nr_nodes = dflow.get_var("ndxi")  # number of 1d cells
+        nr_nodes2d = dflow.get_var("ndx2d")  # number of 2d cells
+        return int(nr_nodes - nr_nodes2d)
+
+    @classmethod
+    def get_waterlevels_1d(cls, dflow: BMIWrapper) -> NDArray[np.float_]:
+        nr_nodes_1d = cls.get_number_1d_nodes(dflow)
+        if nr_nodes_1d == 0:
+            return np.empty(shape=(0), dtype=np.float_)
```

Comment:
Why not return `None` for this case?

Reply:
because of mypy

Reply:
You can make the return type `Optional[NDArray[np.float_]]`.

Import it from `from typing import Optional`

Reply:
done